lib/menu.csv

```diff
@@ -0,0 +1,5 @@
+pizza_1,1
+pizza_2,2
+pizza_3,3
+pizza_4,4
+pizza_5,5
```

Comment:
Nice use of a csv file to contain the menu!

Reply:
and what a great menu it is! fancy a Pizza_3?